hasher-matcher-actioner/terraform/api/main.tf

```diff
@@ -104,77 +104,6 @@ resource "aws_iam_role_policy_attachment" "api_root" {
   policy_arn = aws_iam_policy.api_root.arn
 }
 
-
-# Authorizer API Lambda
-
```

Comment:
Note that I didn't delete the python source for the authorizer... sort of on purpose. was thinking to ask for comments on that.

Reply:
We decided to delete api_auth.py.